src/Template/Element/search_bar.ctp

```diff
@@ -28,17 +28,51 @@
 use Cake\Core\Configure;
 
 $searchQuery = h(str_replace('{{', '\{\{', json_encode($searchQuery)));
-?>
 
-<md-toolbar ng-controller="SearchBarController as ctrl" class="search_bar md-whiteframe-1dp md-primary">
-<?php
 if ($selectedLanguageFrom == null) {
     $selectedLanguageFrom = 'und';
 }
 
 if ($selectedLanguageTo == null) {
     $selectedLanguageTo = 'und';
 }
+?>
+
+<md-toolbar id="search-bar" ng-cloak>
+    <div class="md-toolbar-tools">
+    <?php
+    echo $this->Form->create('Sentence', [
+        'layout' => 'column',
+        'url' => ['controller' => 'sentences', 'action' => 'search'],
+        'type' => 'get',
+        'flex' => '',
+        'ng-cloak' => '',
+        'ng-controller' => 'SidenavController'
+    ]);
+    ?>
+
+    <div layout="row" layout-align="center center" flex>
+        <md-input-container class="md-accent" flex md-no-float>
+            <md-icon>search</md-icon>
+            <input name="query" 
+                accesskey="4" 
+                dir="auto" 
+                ng-model="ctrl.searchQuery" 
+                ng-init="ctrl.searchQuery = <?= $searchQuery ?>"
+                placeholder="<?= __('Search') ?>"/>
```

Comment:
About the string "Search": you should add a context with `__x()` and a translators’ note like below.